src/reporting/clients/IReportsClient.ts

```diff
@@ -146,4 +146,61 @@ export interface IReportsClient {
     reportId: string,
     reportMappingId: string
   ): Promise<Response>;
+
+  /**
+   * Gets all Report Aggregations for a Report.
```

Comment:
Keep the documentation consistent with what is there for the matching methods for Mappings.